apps/src/templates/BaseDialog.jsx

```diff
@@ -87,6 +87,10 @@ export default class BaseDialog extends React.Component {
         ...bodyStyle,
         height: '80%'
       };
+      modalBodyStyle = {
+        ...modalBodyStyle,
+        boxSizing: 'border-box'
+      };
```

Comment:
Note to self: I'm _pretty_ sure this won't cause issues anywhere else (in fact, it might fix things up in a few places) but I should carefully check the surface area of this change and test everything.

Reply:
Checking the full surface area:

The only uses of `BaseDialog` where it's possible that `fullHeight` is true and `useUpdatedStyles` is false are:

  - PreviewModal
  - Snapshot
  - VisualizerModal - this is the component we're intentionally changing.
  - Dialog
    Passes arbitrary props through, so we have to check all of its usages.  The only ones that might be affected are:
    - ImportProjectDialog (another pass-thru)
      - _No affected usages._
    - ImportScreensDialog (another pass-thru)
      - _No affected usages._
    - ConfirmDeleteButton (another pass-thru)
      - _No affected usages._

So we want to check the `Snapshot` and `PreviewModal` components.

### Snapshot

This one looks okay to me, in terms of the box-sizing adjustment. I do notice that the chart isn't sized quite like I'd expect and the key is cut off now, so I should probably go back and check whether these sizing changes affected the snapshot feature in unexpected ways.

![image](https://user-images.githubusercontent.com/1615761/79402403-930da180-7f40-11ea-8752-5d796e51f2a5.png)

### PreviewModal

Again, I'm feeling fine about the box-sizing adjustment here.  Looks like we might want to revisit this and make sure the scrollable area uses the full available height though.

![image](https://user-images.githubusercontent.com/1615761/79402509-ed0e6700-7f40-11ea-8627-42f561ec5070.png)